.github/workflows/Build and test.yml

```
@@ -43,71 +43,23 @@ jobs:
-        uses: jurplel/install-qt-action@v3
-        with:
-          version: 6.6.*
-          host: 'mac'
-          target: 'desktop'
-          arch: 'clang_64'
+      # - name: Install conan
+      #   run: pipx run conan install .
 
       - name: Configure CMake
         env:
           CC: ${{ matrix.compiler_c }}
```

actionlint (1.7.7)

property "compiler_c" is not defined in object type {}

(expression)